services/records/record_templates/models.go

```diff
@@ -0,0 +1,35 @@
+package record_templates
+
+import (
+	"github.com/leanspace/terraform-provider-leanspace/helper/general_objects"
+)
+
+type RecordTemplate struct {
+	ID             string                     `json:"id"`
+	Name           string                     `json:"name"`
+	Description    string                     `json:"description"`
+	RecordState    string                     `json:"state"`
```

Comment:
why the difference in naming? (why `RecordState` on the struct but `state` on the api?

Reply:
It's just the internal name in the back-end service. I've updated it to just be state.